### ISSUE TITLE
Vader Crew still locking up

### ISSUE THREAD

**Issue by drail14me** [Sandrem/FlyCasual#1298]
Just used Vader crew against Wedge. Wedge was tokenless after bumping. I saw the shield come off after using Vader but then the game locked. Screen capture below.

![screen shot 2018-10-25 at 12 17 09 pm](https://user-images.githubusercontent.com/38188214/47518356-64819f80-d850-11e8-884b-3100a3d157aa.png)

**Comment by Sandrem** [comment]
Yes, but I found few problems in code of this ability and fixed them. So, I don't accept new reports about this bug until release of new version.

**Comment by drail14me** [comment]
Awesome! Thanks for your hard work. 

Did you get my message via the FFG forum about any X-Wing tournaments in the Ukraine area? Gold Squadron Podcast wants to travel to stream some other events. I’d love to come with them and stream a tournament in the Ukraine. 

Dennis

**Comment by Sandrem** [comment]
Oh, yes, now I see that message.

As I know, no tournaments are planned in Ukraine, and I don't plan to visit tournaments in another countries in the nearest future. I will tell you if situation will be changed.

You're awesome team, I watched your streams a lot. These streams helped me a lot to understand the game and to win few local tournaments =)

By the way, did you think to stream Fly Casual? Or make a video-review? I know that the game is still in development and has bugs (network mode is still not ready for tournaments). But if more people know about developement - then I can have more support (coders, 3d-developers, testers, patreon supportes), and this can improve speed of development and quality of the game a lot.

**Comment by Sandrem** [comment]
I can add some things to Fly Casual, like custom playmat with name of your team, avatar, or show a simple way to change title (instead of "Test pilot", for example, "Gold Squadron Podcast"/"Gold Squadron Pilot").

**Comment by drail14me** [comment]
I’m not a member of Gold Squadron but am a Patreon of theirs. I will suggest to Dion to do some streams from Fly Casual. Great idea!

Dennis Daniels

**Comment by Sandrem** [comment]
Thank you)

**Comment by drail14me** [comment]
I’ve contacted Dion Morales with Gold Squadron Podcast. He is interested in streaming some Fly Casual games and has asked me to tag him in on an email to you so the two of you can coordinate. What is the best email address for me to use for Dion to contact you?

Looking forward to seeing y’all on stream!

Dennis Daniels

**Comment by Sandrem** [comment]
You can use this e-mail to contact me: oogoeroo@gmail.com
